lib/imapoptions

```diff
@@ -633,6 +633,10 @@ Blank lines and lines beginning with ``#'' are ignored.
    If not set (the default), the value of the "servername" option will
    be used.*/
 
+{ "caldav_send_partstat_updates", 0, SWITCH, "UNRELEASED" }
+/* If enabled, when an attendee changes their participation status,
+   an update will be sent to all other attendees. */
```

Comment:
@rsto This isn't about replies per se.  Its about sending out requests to the other attendees when one of them changes their partstat.  For example, let's say Organizer A invites Attendees B, C, D.  B replies to O saying that they are attending. Updated requests, including B's partstat, are then sent to C and D.  If you have a long list of attendees, this can lead to each attendee receiving a large number of emails as they reply.  This option would prevent the partstat updates being sent to every attendee.

Reply:
got it